tests/unit/generateMonthMatrixTest.ts

```diff
@@ -125,13 +127,6 @@ describe('generateMonthMatrix', () => {
         expect(() => generateMonthMatrix(-1, 0)).toThrow();
     });
 
-    it('throws an error if year or month is not a number', () => {
-        expect(() => generateMonthMatrix()).toThrow();
-        expect(() => generateMonthMatrix(2022, 'invalid')).toThrow();
-        expect(() => generateMonthMatrix('2022', '0')).toThrow();
-        expect(() => generateMonthMatrix(null, undefined)).toThrow();
-    });
```

Comment:
removing that test case since it will be covered by typescript when `generateMonthMatrix` will be migrated